lib/vagrant-service-manager/binary_handlers/binary_handler.rb

```diff
@@ -100,10 +100,9 @@ def handle_windows_binary_path
           oc_version = CDKOpenshiftBinaryHandler::LATEST_OC_VERSION
         end
 
-        unless @options['--cli-version'] != oc_version || @options['--path']
-          path = PluginUtil.fetch_existing_oc_binary_path_in_windows
-          @path = path unless path.nil?
-        end
+        return if @options['--cli-version'] != oc_version || @options['--path']
```

Comment:
@hferentschik , Code looks fine. 
The idea is look for windows existing paths and set if found like `C:\DevSuite\bin\oc.exe`. 

The path should not be set if the user provided `--cli-version` is different than pinned oc version or user mentioned `--path` option.